types/iso15118_charger.yaml

```diff
@@ -126,17 +150,17 @@ types:
       - EVSEMaximumVoltageLimit
     properties:
       EVSEMaximumCurrentLimit:
-        description: "[A] Maximum current the EVSE can deliver"
+        description: Maximum current the EVSE can deliver in A
         type: number
         minimum: -10000
         maximum: 10000
       EVSEMaximumPowerLimit:
-        description: "[W] Maximum power the EVSE can deliver"
+        description: Maximum power the EVSE can deliver in W
         type: number
         minimum: -1000000
         maximum: 1000000
       EVSEMaximumVoltageLimit:
-        description: "[V] Maximum voltage the EVSE can deliver"
+        description: Maximum voltage the EVSE can deliver in V
```

Comment:
This might be personal preference, but I would put the unit closer to its *context*, meaning:
`Maximum voltage (in V), the EVSE can deliver`

Reply:
It's a matter of taste. But I would leave it as it is because of the upcoming merge. The main thing is that the unit is there